lib/app.rb

```diff
@@ -24,6 +25,7 @@
 require_relative 'version'
 require_relative 'app/models'
 require_relative 'app/utils/terminology'
+require '/Users/rscalfani/Documents/code/inferno/inferno-saner/generator/mcode/mcode_generator'
```

Comment:
this won't work on anyone else's machine.  Use a `require_relative` here to avoid reference to you as a user.

Reply:
Also I don't think this `require` is necessary at all now that I think about it